src/freenet/clients/http/WelcomeToadlet.java

```diff
@@ -576,6 +576,10 @@ public void handleMethodGET(URI uri, HTTPRequest request, ToadletContext ctx) th
                 HTMLNode restartForm = ctx.addFormChild(versionContent, ".", "restartForm");
                 restartForm.addChild("input", new String[]{"type", "name"}, new String[]{"hidden", "restart"});
                 restartForm.addChild("input", new String[]{"type", "name", "value"}, new String[]{"submit", "restart2", l10n("restartNode")});
+                // add autostart info for Linux (where there’s no tray *and* there is a defined way for autostart).
+                if (FileUtil.OperatingSystem.Linux == FileUtil.detectedOS) {
```

Comment:
This is incredibly weird to read, because the part that is variable is usually on the left; `x == 5` is way better to read than `5 == x`. 🙂